static/app/views/starfish/components/slideOverPanel.tsx

```diff
@@ -12,21 +11,22 @@ const PANEL_WIDTH = '50vw';
 type SlideOverPanelProps = {
   children: React.ReactNode;
   collapsed: boolean;
+  onOpen?: () => void;
 };
 
 export default forwardRef(SlideOverPanel);
 
 function SlideOverPanel(
-  {collapsed, children}: SlideOverPanelProps,
+  {collapsed, children, onOpen}: SlideOverPanelProps,
   ref: ForwardedRef<HTMLDivElement>
 ) {
   const {query} = useLocation();
   const organization = useOrganization();
   useEffect(() => {
-    if (!collapsed) {
-      trackAnalytics('starfish.panel.open', {organization});
+    if (!collapsed && onOpen) {
+      onOpen();
     }
-  }, [query, collapsed, organization]);
+  }, [query, collapsed, organization, onOpen]);
```

Comment:
Shouldn't we trigger `onOpen` only when state goes from `uncollapsed` to `collapsed`. Seems a bit odd to me to include `query and organization` here.

I think you can remove those as it was only needed for analytic purposes.

Reply:
Just wanted to make sure, you mentioned above that the `query` in the dependency array is to ensure that when you click on another endpoint span summary it will trigger the analytics. Here you said we can get rid of it. I was going to get rid of it but just wanted to double check before doing so.

Reply:
Yeah you should get rid of it.
We removed the ability to click an endpoint to change the sidebar so should be good to remove.